silkworm/silkrpc/http/request_parser_test.cpp

```diff
@@ -37,10 +38,10 @@ TEST_CASE("parse", "[silkrpc][http][request_parser]") {
             const auto result{parser.parse(req, buffer.data(), buffer.data() + bytes_read)};
             CHECK(result == RequestParser::ResultType::bad);
         }
-    }
+    }  // namespace silkworm::rpc::http
```

Comment:
I suspect this is a typo because this curly brace does _not_ close the namespace

Reply:
ok